run_unittests.py

```diff
@@ -1179,6 +1179,15 @@ def assertPathEqual(self, path1, path2):
         '''
         self.assertEqual(PurePath(path1), PurePath(path2))
 
+    def assertPathListEqual(self, pathlist1, pathlist2):
+        self.assertEqual(len(pathlist1), len(pathlist2))
+        worklist = list(zip(pathlist1, pathlist2))
+        for i in worklist:
+            if i[0] == None:
```

Comment:
__[Flake8]__ <sub>posted by Sider</sub>

[E711] comparison to None should be 'if cond is None:'